charts/prometheus-elasticsearch-exporter/templates/deployment.yaml

```
@@ -184,11 +184,11 @@ spec:
       {{- end }}
       {{- with .Values.nodeSelector }}
       nodeSelector:
-        {{- toYaml . | nindent 8 }}
+        {{- tpl (toYaml . | nindent 8) $ }}
```

I'd suggest to indent the block after applying the function on the value: 

```suggestion
        {{- tpl (toYaml .) $ | nindent 8 }}
```